test/test_misc.jl

```diff
@@ -165,7 +165,7 @@ for T in (Float32, Float64), dim in (1,2,3), order in (1,2,4), TensorType in (Te
             @test t^2 ≈ t ⋅ t
             @test t^3 ≈ t ⋅ t ⋅ t
             @test t^-1 ≈ inv(t)
-            @test t^-2 ≈ inv(t)^2
+            @test t^-2 ≈ inv(t) ⋅ inv(t)
```

Comment:
That won't parse correctly then for literal powers.

```julia
@inferred As^3
ERROR: use `⋅` (`\cdot`) for single contraction and `⊡` (`\boxdot`) for double contraction instead of `*`
Stacktrace:
 [1] power_by_squaring(::Tensors.SymmetricTensor{2,3,Float64,6}, ::Int64) at ./intfuncs.jl:182
 [2] ^(::Tensors.SymmetricTensor{2,3,Float64,6}, ::Int64) at ./linalg/dense.jl:326
```
but can add some `isa`